addons/website/static/src/js/menu/content.js

```diff
@@ -271,6 +277,26 @@ var PagePropertiesDialog = widget.Dialog.extend({
             id: m[2] | 0,
         };
     },
+    /**
+     * Converts a string representing the browser datetime
+     * (exemple: Albanian: '2018-Qer-22 15.12.35.')
+     * to a string representing UTC in Odoo's datetime string format
+     * (exemple: '2018-04-22 13:12:35').
+     *
+     * The time zone of the datetime string is assumed to be the one of the
+     * browser and it will be converted to UTC (standard for OpenERP 6.1).
+     *
+     * @private
+     * @param {String} value A string representing a datetime.
+     * @returns {String} A string representing an UTC datetime.
```

Comment:
{String|false} A string representing an UTC datetime is the given value is valid, false otherwise.